guides/FRONTEND.md

```diff
@@ -0,0 +1,203 @@
+## MetaGame's FrontEnd API Guide
+
+### Requirements
+
+In order to develop MetaGame FrontEnd components, you need.
+
+- Node.js
+
+- Docker
+
+You will need to be able to start the backend with:
+
+```bash
+yarn docker:start
+```
+
+If you have difficulties running the backend, checkout the [Backend Guide](BACKEND.md).
```

Comment:
We dont need them to run the backend, they can just set the `NEXT_PUBLIC_GRAPHQL_URL` in their .env file to `https://hasura-7s6e.onrender.com/`